### PR TITLE
Add Nov 13

### PR DESCRIPTION
**Pull request by blackboxsw**

### CODE REVIEW

File: doc/2018/2018-11-13.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight: Ubuntu 19.04 (Disco Dingo) archive is open for development!
+As [announced](https://lists.ubuntu.com/archives/ubuntu-announce/2018-October/000237.html), Cosmic Cuttlefish was released was released Oct 18.  Check out the [release notes](https://wiki.ubuntu.com/CosmicCuttlefish/ReleaseNotes) for more information on changes, download the latest [server ISO](http://releases.ubuntu.com/18.10/), or try out our [subiquity server installer](http://cdimage.ubuntu.com/ubuntu-server/daily-live/current/). As of Friday last week, the Ubuntu archive is now officially open for development on Disco Dingo. See the [Disco Dingo release schedule](https://wiki.ubuntu.com/DiscoDingo/ReleaseSchedule) for timing of this upcoming release. Let's continue to make Ubuntu excellent!
```
Comment (panlinux) [inline doc/2018/2018-11-13.md]: "was released" twice.

Comment (panlinux) [reply doc/2018/2018-11-13.md]: suggestion: replace "for timing of this upcoming release" with "for important milestones of this upcoming release"

Comment (blackboxsw) [reply doc/2018/2018-11-13.md]: fixed both. Thanks.

**Review by panlinux — APPROVED** [review]
+1

**Pull request merged**